src/js/options.js

```diff
@@ -13,17 +13,23 @@ document.addEventListener('DOMContentLoaded', () => {
     settings: {
       theme: 'light',
       addContextMenu: true,
+      addPageAction: true,
       animateItems: !isFirefox,
       openNewTab: false,
       viewAll: true
     }
   }
 
+  if (!isFirefox) {
```

Comment:
This is awesome! I really like this good job!

Add comment so we know why it is there in the future. something like:
`// Icon in address bar only for firefox does not work in chrome`